### ISSUE TITLE
Unable to parse concepts list

### ISSUE THREAD

**Issue by mykeehu** [d8ahazard/sd_dreambooth_extension#243]
## Kindly read this ENTIRE form and fill it out with the requested information. It may seem tedious, but it's even more tedious asking basic questions to ensure I'm not trying to troubleshoot a bug I've already fixed. Help me help you, answer all the questions and follow the steps below before posting.

**Please find the following lines in the console and paste them below. _If you do not provide this information, your issue will be automatically closed._**

`
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: 98947d173e3f1667eba29c904f681047dea9de90
Installing requirements for Web UI
Skipping dreambooth installation.
Dreambooth revision is 725d85ca5485246a430d2f31f624646d061e8104
Diffusers version is 0.7.2.
Torch version is 1.12.1+cu116.
Torch vision version is 0.13.1+cu116.
`

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes

**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes

**Have you updated Dreambooth to the latest revision?**
Yes

**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Yes

**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this. Reply 'OK' Below to acknowledge that you did this.**
Ok

**Describe the bug**

I wanted to continue after the end of the training because I was not satisfied with the result. That's when I got this error that prevented me from continuing the training:

Trying to parse: [{"instance_prompt": "a photo of ooosggg woman", "class_prompt": "a photo of a woman", "sample_prompt": None, "negative_prompt": None, "instance_data_dir": "H:\\Stable-Diffusion-Automatic\\Dreambooth\\ooosggg", "class_data_dir": "V:\\Programok\\!AI cuccok\\Stable Diffusion web UI - Automatic1111\\Dreambooth cuccok\\Templates for training\\Class-Regularization images 2 - JoePenna\\woman_dimm"}]
Unable to load concepts as JSON, trying as file: Expecting value: line 1 column 105 (char 104)
Loaded concepts from [{'instance_prompt': 'a photo of ooosggg woman', 'class_prompt': 'a photo of a woman', 'sample_prompt': None, 'negative_prompt': None, 'instance_data_dir': 'H:\\Stable-Diffusion-Automatic\\Dreambooth\\ooosggg', 'class_data_dir': 'V:\\Programok\\!AI cuccok\\Stable Diffusion web UI - Automatic1111\\Dreambooth cuccok\\Templates for training\\Class-Regularization images 2 - JoePenna\\woman_dimm'}]
Trying to parse: None
Unable to parse concepts list.

**Provide logs**

If a crash has occurred, please provide the *entire* stack trace from the log, including the last few log messages *before* the crash occurred.

**Environment**

What OS? Windows

If Windows - WSL or native? Native

What GPU are you using? RTX 3060 12 GB

**Screenshots/Config**
If the issue is specific to an error while training, please provide a screenshot of training parameters or the db_config.json file from /models/dreambooth/MODELNAME/db_config.json

{
    "max_token_length": "75",
    "model_dir": "H:\\Stable-Diffusion-Automatic\\stable-diffusion-webui\\models\\dreambooth\\ooosggg",
    "pretrained_model_name_or_path": "H:\\Stable-Diffusion-Automatic\\stable-diffusion-webui\\models\\dreambooth\\ooosggg\\working",
    "class_guidance_scale": 7.5,
    "class_negative_prompt": "",
    "class_infer_steps": 60,
    "model_name": "ooosggg",
    "scheduler": "ddim",
    "src": "model-v1-5-pruned-emaonly.ckpt [81761151]",
    "total_steps": 0,
    "revision": 3200,
    "pretrained_vae_name_or_path": null,
    "instance_data_dir": "H:\\Stable-Diffusion-Automatic\\Dreambooth\\ooosggg",
    "class_data_dir": "V:\\Programok\\!AI cuccok\\Stable Diffusion web UI - Automatic1111\\Dreambooth cuccok\\Templates for training\\Class-Regularization images 2 - JoePenna\\woman_dimm",
    "instance_prompt": "a photo of ooosggg woman",
    "class_prompt": "a photo of a woman",
    "save_sample_prompt": null,
    "save_sample_negative_prompt": null,
    "n_save_sample": null,
    "seed": null,
    "save_guidance_scale": null,
    "save_infer_steps": null,
    "with_prior_preservation": true,
    "num_class_images": 480,
    "output_dir": "H:\\Stable-Diffusion-Automatic\\stable-diffusion-webui\\models\\dreambooth\\ooosggg",
    "resolution": 512,
    "center_crop": false,
    "train_text_encoder": true,
    "train_batch_size": 1,
    "sample_batch_size": 1,
    "num_train_epochs": 80,
    "max_train_steps": 800,
    "gradient_accumulation_steps": 1,
    "gradient_checkpointing": true,
    "learning_rate": 1e-06,
    "scale_lr": true,
    "lr_scheduler": "polynomial",
    "lr_warmup_steps": 320,
    "use_8bit_adam": true,
    "adam_beta1": 0.9,
    "adam_beta2": 0.999,
    "adam_weight_decay": 0.01,
    "adam_epsilon": 1e-08,
    "max_grad_norm": 1,
    "save_preview_every": 0,
    "save_embedding_every": 200,
    "mixed_precision": "fp16",
    "not_cache_latents": true,
    "concepts_list": null,
    "use_cpu": false,
    "pad_tokens": true,
    "hflip": false,
    "use_ema": true,
    "prior_loss_weight": 1,
    "tokenizer_name": null
}

**Comment by mykeehu** [comment]
""concepts_list": null," but the data was previously included here when you saved. It seems it will not be saved here in the new version?

**Comment by d8ahazard** [comment]
> `"concepts_list": null,` but the data was previously included here when you saved. It seems it will not be saved here in the new version? You should create this block when saving, as in older versions: `"concepts_list": [ { "instance_prompt": "a photo of ooosggg woman", "class_prompt": "a photo of a woman", "sample_prompt": "ooosggg", "negative_prompt": "", "instance_data_dir": "H:\\Stable-Diffusion-Automatic\\Dreambooth\\ooosggg", "class_data_dir": "V:\\Programok\\!AI cuccok\\Stable Diffusion web UI - Automatic1111\\Dreambooth cuccok\\Templates for training\\Class-Regularization images 2 - JoePenna\\woman_dimm" } ],`

I've got "refining the concept list creation" in the stuff I'm currently testing. Should be a non-issue after I push the next update later today.

**Comment by mykeehu** [comment]
I reverted back to e66b34b925ff3ff1aafa70b0ab08d95917cc4791 because I entered the parameters in json in vain, it let me continue once, then at the end of the workout if I start or continue a new workout, I get CUDA OOM errors. Something went wrong in the patch for version 30a4385d54904622efddff3123fabcffffd04d41f.

**Comment by mykeehu** [comment]
The continuation is solved, but without xformers I keep getting OOM errors. The WebUI is using 3 GB of VRAM foplyamatically and I don't understand why. Memory cleanup is fine, but as soon as VAE and text encoder load, it jumps back to 3 GB. I've tried Half switch, the other parameters are the same as I used yesterday. I cannot train on 12 GB without xformers with Scale Learning Rate. :(

**Comment by mykeehu** [comment]
I closed this issue, problem solved in latest builds.
